torchvision/prototype/transforms/functional/_meta.py

```diff
@@ -367,7 +367,7 @@ def convert_dtype_image_tensor(image: torch.Tensor, dtype: torch.dtype = torch.f
     else:
         # int to float
         if float_output:
-            return image.to(dtype).div_(_FT._max_value(image.dtype))
+            return image.to(dtype).mul_(1.0 / _FT._max_value(image.dtype))
```

Comment:
Shall we revert this since it's not beneficial?

Reply:
Lol, didn't notice that the columns were switched and that there is no performance issue here. I think I need another coffee. 😄